test/Dask/test_daskio.py

```diff
@@ -0,0 +1,70 @@
+import numpy
+import dask
+import dask.array as da
+from dask.distributed import Client
+
+from lyncs_utils import prod
+from lyncs_io.dask_io import DaskIO
+import lyncs_io as io
+
+from lyncs_io.testing import domain_loop, workers_loop, chunksize_loop, tempdir
+
+
+@domain_loop
+@chunksize_loop
+@workers_loop
+def test_daskio_load(tempdir, domain, chunksize, workers):
+
+    client = Client(n_workers=workers, threads_per_worker=1)
```

Comment:
To avoid opening and closing clients, you can initialize one on `max_workers` using a fixture with scope session
```
@fixture(scope="session")
def client():
    client = Client(n_workers=max_workers, threads_per_worker=1)
    yield client
    client.shutdown()
```
and then limit the number of workers at the compute level, `compute(num_workers=workers)`